test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/transforms/DeferredFieldIsRenamedTestSnapshot.kt

```diff
@@ -55,7 +55,8 @@ public class DeferredFieldIsRenamedTestSnapshot : TestSnapshot() {
                     |         "defer"
                     |       ],
                     |       "data": {
-                    |         "overallString": "string for the deferred renamed field"
```

Comment:
Because we didn't clone the result before Nadel mutated it, the test snapshots wrote down the final result that already had the field renamed.